docs/source/installation_instructions.rst

```diff
@@ -0,0 +1,91 @@
+.. The purpose of this file is to provide instructions on how to install the software necessary to access and run
+   the pytentiostat software package.
+
+Installation Instructions
+=========================
+
+.. seealso::
+   Instructions are listed below for installing Miniconda, but the full user documentation for Anaconda is available at
+   https://conda.io/projects/conda/en/latest/user-guide/install/index.html
+
+System Requirments
+------------------
+
+Before installing Anaconda, the following system requirements must be met:
+
+* 32- or 64-bit computer
+* 400 MB disk space
+* Windows, MacOS, or Linux
+* Python 2.7, 3.4, 3.5, 3.6, or 3.7
+* pycosat
+* PyYaml
+* Requests
+
+If you do not already have one of the required versions of Python, follow the instructions below.
+
+.. note::
+
+   We used Python 3.7 for the Pytentiostat software, so it is encouraged to download that version to ensure
+   compatibility.
+
+Python Installation
+--------------------
+
+Go to the following page that corresponds with the operating system that you are working on.
+
+* Windows:      https://www.python.org/downloads/windows/
+* MacOS:        https://www.python.org/downloads/mac-osx/
+* Linux/UNIX:   https://www.python.org/downloads/source/
+
+Click on the download link for the latest release of Python 3.7. Double click the .exe file and follow prompts to proceed
+with installation.
+
+Install pycosat, PyYaml, and Requests.
```

Comment:
These should be in the requirements list, then they will automatically install with the conda install of pytentiostat, so this should also be deleted from here

Reply:
This confuses me a bit because the installation instructions in the miniconda docs say that these are system requirements, implying that it all needs to be installed prior to miniconda. Maybe that's not the case? Therefore, all the users should have to do is click the link to install the python 3 version of miniconda for their operating system.